src/ripple/app/tx/impl/apply.cpp

```diff
@@ -70,6 +70,45 @@ invoke_preflight (PreflightContext const& ctx)
 }
 
 /*
+invoke_calculateBaseFee<T> uses name hiding to accomplish
+compile-time polymorphism of (presumably) static
+class functions for Transactor and derived classes.
+*/
+template<class T>
+static
+std::uint64_t
+invoke_calculateBaseFee(PreclaimContext const& ctx)
+{
+    return T::calculateBaseFee(ctx);
+}
+
+static
+std::uint64_t
+invoke_calculateBaseFee(PreclaimContext const& ctx)
+{
+    switch (ctx.tx.getTxnType())
+    {
+    case ttACCOUNT_SET:     return invoke_calculateBaseFee<SetAccount>(ctx);
```

Comment:
I'd prefer `SetAccount::calculateBaseFee(ctx);` over `invoke_calculateBaseFee<SetAccount>(ctx)`. On less level of indirection and does the same thing.